api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8946,4 +8946,61 @@
             <column name="address15" type="varchar(255)"></column>
         </addColumn>
     </changeSet>
+
+
+    <changeSet id="TRUNK-3422-adding-View-Visits-Privilege" author="PralayRamteke" >
+            <preConditions  onFail="MARK_RAN">
+                <not>
+                    <sqlCheck expectedResult="0">
+                    SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
+                    </sqlCheck>
+                </not>
+            </preConditions>
+            <comment>
+                Add "View Visits" privilege to the roles having "View Encounter"
+            </comment>
+                <sql>
+                    INSERT INTO privilege (privilege,description)
```

Comment:
@puevigreven after having looked at PrivilegeConstants, "View" needs to be changed to "Get".